torch/autograd/profiler_util.py

```diff
@@ -495,7 +495,8 @@ def __repr__(self):
         return (
             '<FunctionEvent id={} name={} device_type={} node_id={} cpu_time={} start_us={} end_us={} '
             'cpu_children={} cuda_time={} name={} thread={} input_shapes={} '
-            'cpu_memory_usage={} cuda_memory_usage={} is_async={} is_remote={} seq_nr={} is_legacy={}>'.format(
+            'cpu_memory_usage={} cuda_memory_usage={} '
```

Comment:
Is this changed?

Reply:
Good catch, I had made a change then removed it.  I'm reverting back to the original, so this file will be unchanged.